lib/broccoli/ember-app.js

```diff
@@ -1040,34 +1044,6 @@ class EmberApp {
     });
   }
 
-  /**
-    @private
-    @method _processedBowerTree
-    @return
-  */
-  _processedBowerTree() {
-    if (!this._cachedBowerTree) {
-      // do not attempt to merge bower and vendor together
```

Comment:
these comments might make sense to come along with the change.

Reply:
good point.